{
  "name": "CVE-2021-3426",
  "seq": "2021-3426",
  "type": "CAN",
  "status": "Candidate",
  "phase": {
    "text": "Assigned",
    "date": "20210309"
  },
  "desc": "There's a flaw in Python 3's pydoc. A local or adjacent attacker who discovers or is able to convince another local or adjacent user to start a pydoc server could access the server and use it to disclose sensitive information belonging to the other user that they would not normally be able to access. The highest risk of this flaw is to data confidentiality. This flaw affects Python versions before 3.8.9, Python versions before 3.9.3 and Python versions before 3.10.0a7.",
  "refs": [
    {
      "text": "https://security.netapp.com/advisory/ntap-20210629-0003/",
      "source": "CONFIRM",
      "url": "https://security.netapp.com/advisory/ntap-20210629-0003/"
    },
    {
      "text": "FEDORA-2021-067c9deff1",
      "source": "FEDORA",
      "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QNGAFMPIYIVJ47FCF2NK2PIX22HUG35B/"
    },
    {
      "text": "FEDORA-2021-0a8f3ffbc0",
      "source": "FEDORA",
      "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/BF2K7HEWADHN6P52R3QLIOX27U3DJ4HI/"
    },
    {
      "text": "FEDORA-2021-1769a23935",
      "source": "FEDORA",
      "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/DQYPUKLLBOZMKFPO7RD7CENTXHUUEUV7/"
    },
    {
      "text": "FEDORA-2021-2ab6f060d9",
      "source": "FEDORA",
      "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/VPX7Y5GQDNB4FJTREWONGC4ZSVH7TGHF/"
    },
    {
      "text": "FEDORA-2021-a26257ccf5",
      "source": "FEDORA",
      "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/LM5V4VPLBHBEASSAROYPSHXGXGGPHNOE/"
    },
    {
      "text": "FEDORA-2021-a311bf10d4",
      "source": "FEDORA",
      "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/25HVHLBGO2KNPXJ3G426QEYSSCECJDU5/"
    },
    {
      "text": "FEDORA-2021-b6b6093b3a",
      "source": "FEDORA",
      "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/N6VXJZSZ6N64AILJX4CTMACYGQGHHD5C/"
    },
    {
      "text": "GLSA-202104-04",
      "source": "GENTOO",
      "url": "https://security.gentoo.org/glsa/202104-04"
    },
    {
      "text": "https://bugzilla.redhat.com/show_bug.cgi?id=1935913",
      "source": "MISC",
      "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1935913"
    },
    {
      "text": "https://www.oracle.com/security-alerts/cpujan2022.html",
      "source": "MISC",
      "url": "https://www.oracle.com/security-alerts/cpujan2022.html"
    },
    {
      "text": "https://www.oracle.com/security-alerts/cpuoct2021.html",
      "source": "MISC",
      "url": "https://www.oracle.com/security-alerts/cpuoct2021.html"
    },
    {
      "text": "[debian-lts-announce] 20210405 [SECURITY] [DLA 2619-1] python3.5 security update",
      "source": "MLIST",
      "url": "https://lists.debian.org/debian-lts-announce/2021/04/msg00005.html"
    },
    {
      "text": "[debian-lts-announce] 20230630 [SECURITY] [DLA 3477-1] python3.7 security update",
      "source": "MLIST",
      "url": "https://lists.debian.org/debian-lts-announce/2023/06/msg00039.html"
    }
  ],
  "votes": {}
}
